### ISSUE TITLE
Crop function work only once

### ISSUE THREAD

**Issue by rakeshpatel09** [prscX/react-native-photo-editor#105]
crop function only work once. if we want to crop picture again then it not work.

**Comment by hamzaouhssain1993** [comment]
yeah this is in Android, i advice you to do not allow a second crop after the first time ( in the native android part you should do it). I fixed for the moment like this

**Comment by memanoj** [comment]
Hi i fix the this issue please check https://github.com/prscX/react-native-photo-editor/issues/190 may hope it will help you so i am closing the issue now so i requested to [prscX](https://github.com/prscX) to check this and update it the git repo it will help to other developer also in Android who are stuck in this issue.
